src/components/controls/panel-toggle.tsx

```diff
@@ -0,0 +1,32 @@
+// A slider toggle to adjust the state of a panel via dispatch.
+
+import React from "react";
+import { useDispatch } from "react-redux";
+
+import Toggle from "./toggle";
+import { togglePanelDisplay } from "../../actions/panelDisplay";
+
+type Props = {
+  panel: string
+  on: boolean
+}
+
+const PanelToggle = ({ panel, on }: Props) => {
+  const dispatch = useDispatch();
+
+  // There is no slider label since the title in the annotated header acts as a
+  // visual label.
+  // FIXME: Add a hidden label?
+
+  return (
+    <Toggle
+      display={true}
+      on={on}
+      callback={() => dispatch(togglePanelDisplay(panel))}
+      label={""}
+      style={{display: "inline"}}
```

Comment:
I tried making prototypes but found myself going back to code changes for the interactive feel, which is an important factor in this change, and something that a static picture can't provide.

Starting with two options: [1](https://auspice-victorlin-move--vboshc.herokuapp.com/ncov/gisaid/global), [2](https://auspice-victorlin-move--vok2df.herokuapp.com/ncov/gisaid/global). The only difference between them is the toggle being visual or textual.

My thoughts that influenced the initial options:

1. The horizontal divider line is generally useful. There shouldn't be a need to indent if the horizontal divider line is there.
2. I like having the header text be the first thing on the left. It's visually more of a "header" to the section contents than if a toggle was on the far left.
3. Having the visual toggle to the right of the text label can be confusing with other visual toggles appearing on the other side. Is this what prompted you to try a textual toggle in your mockup above?
4. It can be good to keep the toggles, whether visual or textual, vertically aligned so that a minimal amount of visual searching and mouse movement is required to interact with multiple toggles.

Reply:
[Cross-posted to Slack](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1696895285086759) with nextstrain.org preview links, including one for #1705.

Reply:
The previews have gone dark, but FWIW I'm happy with the current styles in this PR as it relates to this conversation